arrows/ocv/tests/test_image.cxx

```diff
@@ -345,43 +345,77 @@ TYPED_TEST(image_conversion, vital_to_ocv_interleaved)
 }
 
 // ----------------------------------------------------------------------------
-TEST(image, bgr_to_rgb)
+template <typename T>
+class bgr_conversion : public ::testing::Test
```

Comment:
Can we call this `image_cs_conversion` instead? That way the name is still of the form `ocv:image*`... Otherwise looks good; thanks!

(...and I suppose pedantically, `image_cs_conversion_types`... though TBH the alias isn't strictly necessary; I'm pretty sure `image_conversion_types` exists because the line wrapping is otherwise atrocious.)